cli/cmd/install/install.go

```diff
@@ -287,7 +273,77 @@ timescaledb-single:
 		fmt.Println("skipping the wait for pods to come to a running state because --skip-wait is enabled.")
 	}
 
-	// create the default otelcol CR as operator should be is up & running by now....
+	return nil
+}
+
+func (c *InstallSpec) manageDBSecrets() error {
+	if c.dbURI != "" {
+		helmValues = appendDBURIValues(helmValues)
+	} else {
+		e, err := helmClient.ExportValuesFieldFromChart(c.Ref, c.ConfigFile, []string{"timescaledb-single", "enabled"})
+		if err != nil {
+			return err
+		}
+		enableTimescaleDB, ok := e.(bool)
+		if !ok {
+			return fmt.Errorf("timescaledb-single.enabled was not a bool")
+		}
+
+		// if timescaledb is disabled we do not need
+		// to create DB level secrets
+		if enableTimescaleDB {
+			err = c.createSecrets()
```

Comment:
Why do we need to run `createSecrets` when timescaledb helm chart is creating secrets internally?

Instead of creating secrets directly, can we just generate random credentials and reference them in correct places in promscale and timescaledb charts (values)?